.github/workflows/main.yaml

```diff
@@ -37,15 +66,17 @@ jobs:
           path: reports
           persist-credentials: false
       - name: Archive older reports
+        working-directory: performance/
         shell: bash
-        run: bash .github/archiveReports.sh
+        run: |
+          bash .github/archiveReports.sh
```

Comment:
single line run command - no need of pipe

Reply:
added single line commands